migration/config-repair/remove_incorrect_configs.py

```diff
@@ -0,0 +1,20 @@
+import tables
+from shutil import copyfile
+
+
+def remove_configs(filename, cluster, stationnumber):
+    """
+    Remove all but the first configuration row
+    """
+    
+    copyfile(filename, filename+'_backup')
```

Comment:
Don't backup in your script!

Backup. Check backup. Make sure the backup is untouchable by your script.

Then run the script.